app/ui/client/views/app/room.js

```diff
@@ -890,7 +890,7 @@ Meteor.startup(() => {
 
 			let room = Rooms.findOne({ _id: rid }, { fields: { t: 1 } });
 
-			if (room?.t === 'l' && !FlowRouter.getParam('tab')) {
+			if (room?.t === 'l') {
```

Comment:
@renatobecker I forgot to revert this change in the previous commit. So I'm doing it here

Reply:
Since we don't have a `Chat History` button in contacts directory, this is no longer needed